pkg/tracking/store/sql/tags.go

```diff
@@ -80,3 +80,136 @@ func (s TrackingSQLStore) setTagsWithTransaction(
 
 	return nil
 }
+
+func (s TrackingSQLStore) SetTag(
+	ctx context.Context, runID, key, value string,
+) *contract.Error {
+	if runID == "" {
```

Comment:
Go validations should already have covered this.

Reply:
When I remove that check, the tests seem to fail... How can I make sure that the custom validator I wrote is set up and working as expected? 🤔

Reply:
You could easily first manually check it using `mage dev`. Just as a sanity it is kicking in.

Reply:
This should still not be checked here, your validation covered this.